src/app/constants/apiConstants.ts

```diff
@@ -22,7 +22,7 @@ export const MODEL_ID_REF = '/ref:modelId?';
 export const MODEL_ID = 'modelId=';
 export const API_VERSION = 'api-version=';
 export const AND = '&';
-export const PUBLIC_REPO_HOSTNAME = 'repo.azureiotrepository.com';
+export const PUBLIC_REPO_HOSTNAME = 'devicemodeltest.azureedge.net';
```

Comment:
Point in time?

Reply:
I wouldn't use the test url if this is going to be released. Would recommend `devicemodels.azure.com`

Reply:
This is what I saw in the wiki of model repo. Let me change it. I am also thinking have this pr only approved but not merged, if we need to get anything out before 10/16.